src/main/webapp/itempagescript.js

```diff
@@ -12,15 +12,75 @@ async function loadItemPage() {
       fetch(`/itempagedata?itemId=${itemId}`)
           .then((response) => response.json())
           .then((itemPageData) => {
-            createSelectedItemCard(itemPageData.item);
-            getItemPageComments(itemPageData.comments);
+            fetch('/login')
+                .then((response) => response.json())
+                .then((loginResponse) => {
+                  if (loginResponse.isUserLoggedIn) {
+                    createSelectedItemCard(itemPageData.item);
+                    getItemPageComments(itemPageData.comments);
+                  } else {
+                    createSelectedItemCard(itemPageData.item);
+                    getItemPageComments(itemPageData.comments);
+                    hideCommentBox();
+                    const login = $('#login');
+                    login.append($(
+                        '<a href="' + loginResponse.LoginURL +
+                        '"><button type="button" class="btn btn-dark my-sm-0">Login to Comment</button></a>'));
+                  }
+                })
+                .catch((error) => {
+                  console.log('failed to fetch login status: ' + error);
+                });
           });
     } else {
       console.log('ItemId is empty!');
     }
   });
 }
 
+
+/**
+ * Enables access to the profile if the user is logged in by adding a "Profile"
+ * link and a "Logout" link to the navbar, if the user is not logged in then it
+ * adds a link to login.
+ */
+function initializeNavBarProfileSection() {
```

Comment:
I already have this method defined in dashboard-script.js, you should import it from there instead of copying it here.